app/models/saved_claim/veteran_readiness_employment_claim.rb

```diff
@@ -144,8 +142,20 @@ def upload_to_vbms(doc_type: '1167')
     )
 
     log_to_statsd('vbms') do
-      uploader.upload!
+      response = uploader.upload!
+
+      if response[:vbms_document_series_ref_id].present?
+        updated_form = parsed_form
+        updated_form['documentId'] = response[:vbms_document_series_ref_id]
+        update!(form: updated_form.to_json)
+      end
     end
+
+    send_vbms_confirmation_email(user)
+  rescue
+    log_message_to_sentry('Error uploading VRE claim to VBMS',
+                          :warn, { uuid: user.uuid })
+    send_to_central_mail!(user)
```

Comment:
Something I missed on cleanup, but we should rename this method to something else, such as 'send_to_benefits_intake' or 'send_to_lighthouse' where it's defined.